### PR TITLE
Feat/model without lr flip

### PR DESCRIPTION
**Pull request by stmharry**

### CODE REVIEW

**Comment by stmharry** [comment]
This improved FDI labeling from the modeling stage. Training images are now not flipped left-right, so quadrants I and II won't get confused.
This has the benefit of greatly improving test set raw object detections, and might very slightly degrade validation set performance due to lesser augmentation. Overall this should be a net benefit.

image
